website/data/guides-nav-data.json

```diff
@@ -64,6 +64,14 @@
       {
         "title": "Vault Cluster Monitoring",
         "path": "operations/monitoring"
+      },
+      {
+        "title": "Vault Deployment Guide",
+        "path": "operations/deployment-guide"
+      },
+      {
+        "title": "Performance Standby Nodes",
+        "path": "operations/performance-nodes"
```

Comment:
`/guides` is already hidden / unlinked from main navigation, so rather than add another layer of complexity here, I've just added the missing pages.

Should note as well that all these paths redirect to Learn anyways. But felt that the deletion of `guides` content probably merits a separate PR, and should probably be done all at once for clarity.